### PR TITLE
Support dragging connect icon

### LINKED ISSUE #15: Connect arrow doesn't support drag
### Describe the Problem

Nothing happens on dragging connection icon.

### Expected Behavior

User should be able to "drag start" arrow to connect to other elements.

### PR DESCRIPTION
**Pull request by smbea**
Closes #15

### CODE REVIEW

File: lib/features/contextPad/ImprovedContextPadProvider.js [hunk 1]
```diff
@@ -176,7 +176,7 @@ ImprovedContextPadProvider.$inject = [ 'contextPad', 'eventBus', 'popupMenu' ];
 
 // helpers //////////
 function getIcon(icon) {
-  return `<div class="entry">${icons[icon]}</div>`;
+  return `<div class="entry" draggable="true">${icons[icon]}</div>`;
```
Comment (philippfromme) [inline lib/features/contextPad/ImprovedContextPadProvider.js:179]: Shouldn't we add `draggable="true"` only to the icons that support drag to be semantically correct?

Comment (smbea) [reply lib/features/contextPad/ImprovedContextPadProvider.js]: That makes total sense. Updated

File: lib/features/contextPad/ImprovedContextPadProvider.js [hunk 2]
```diff
@@ -176,6 +176,10 @@ ImprovedContextPadProvider.$inject = [ 'contextPad', 'eventBus', 'popupMenu' ];
 
 // helpers //////////
 function getIcon(icon) {
+  if (icon === 'connect') {
```
Comment (nikku) [inline lib/features/contextPad/ImprovedContextPadProvider.js:179]: Let us not hard-code `draggable` here (based on the `connect` icon), but instead infer whether the icon is draggable from handlers registered with [a particular entry](https://github.com/camunda/improved-canvas/blob/a2b1d352b6ccd69dffb8c955221e76f67a1dbbec/lib/features/contextPad/ImprovedContextPadProvider.js#L103).

This way we know that we _always_ will render the appropriate HTML based on whether a drag handler is registered on the entry or not.

**Review by nikku — CHANGES_REQUESTED** [review]
Cf. https://github.com/camunda/improved-canvas/pull/16/commits/a2b1d352b6ccd69dffb8c955221e76f67a1dbbec#r1432508859

**Review by nikku — APPROVED** [review]

**Pull request merged**